adafruit_vl53l1x.py

```diff
@@ -301,3 +302,13 @@ def _read_register(self, address, length=1):
             i2c.write(struct.pack(">H", address))
             i2c.readinto(data)
         return data
+
+    def set_address(self, new_address):
+        """
+        Set a new I2C address to the instantaited object. This is only called when using
+        multiple VL53L0X sensors on the same I2C bus (SDA & SCL pins). See also the
+        `example <examples.html#multiple-vl53l1x-on-same-i2c-bus>`_ for proper usage.
+        """
+        new_addres_as_byte_string = new_address.to_bytes(1, "little")
+        self._write_register(0x01, new_addres_as_byte_string)
```

Comment:
Please add a new `_I2C_SLAVE_DEVICE_ADDRESS` const for the `0x01` register address, similar to VL54L0X library.

Reply:
Works for me.  Should be good now.